packages/story-editor/src/components/inspector/inspectorProvider.js

```diff
@@ -36,11 +36,12 @@ import { useFeature } from 'flagged';
 import { useAPI } from '../../app/api';
 import { useStory } from '../../app/story';
 import { useHighlights } from '../../app/highlights';
-import { DOCUMENT, DESIGN, PUBLISH_MODAL_DOCUMENT } from './constants';
+import { DOCUMENT, STYLE, PUBLISH_MODAL_DOCUMENT, INSERT } from './constants';
 import Context from './context';
 import DesignInspector from './design';
+import Library from '../library';
 
-const INSPECTOR_TAB_IDS = new Set([DOCUMENT, DESIGN]);
+const INSPECTOR_TAB_IDS = new Set([INSERT, DOCUMENT, STYLE]);
 function InspectorProvider({ inspectorTabs, children }) {
```

Comment:
Regarding naming, I think that we could change the name to just _Sidebar_, and that would work fine.

The only reason I invented the two names, _Library_, and _Inspector_, in the first place was just to have some identifiers. Neither name is used in the UI as far as I remember, so they are only internal anyway.

We did use _Sidebar_ at one point for the flyout menus (such as color picker), but I believe they were all changed to use our popup/modal API, so _Sidebar_ should be a "free" term again now. Note that "inspector" is used as a pre- and postfix for many elements.

We could then also rename the Library to Insert, but I don't think that's necessary. I feel like the name still reflects its purpose.

I do think all the renaming should be moved to a new ticket. Just to keep this PR as clean as it is right now (which is impressively low footprint considering the huge change in UX).

Reply:
big fan of `Sidebar` - i _still_ have to remind myself which is the library and which is the inspector sometimes.

Reply:
> We could then also rename the Library to Insert, but I don't think that's necessary. I feel like the name still reflects its purpose.

Agreed that we could just keep Library as it is, could even keep Design as Design IMO unless Style is generally thought to be a better name. E.g. if the visual name changes again in the future, we'd probably keep the naming in the code the same and only change the visual text, so if Design still reflects what that tab does then seems ok to keep that.